app.js

```diff
@@ -38,14 +39,20 @@ if (app.get("env") === "production") {
 }
 
 app.use(session(sessionParms));
-app.use(require("connect-flash")());
+app.use(require("connect-flash")()); //
+
+app.use(require("./middleware/storeLocals"));
```

Comment:
Not a blocker, but for organization purposes best to consolidate app.use statements towards the top (if you can - the session one on line 41 is a good example of one that needs to be lower)